sdk/src/account_info.rs

```diff
@@ -0,0 +1,99 @@
+use anchor_lang::{prelude::Result, AnchorDeserialize, AnchorSerialize};
```

Comment:
I was thinking about kicking out `anchor_lang::Result` entirely and defining our own error enum with `thiserror::Error` and then make it convertable to `solana_program::ProgramError`.

I didn't want to do it in this PR, because I think it will be long and intrusive change. I'm happy to do it when in a PR dedicated to Anchor-less program support.